gradle-plugin/it/src/it/triggersannotation/expected/kubernetes.yml

```diff
@@ -9,7 +9,6 @@ items:
       jkube.eclipse.org/git-url: "@ignore@"
       jkube.eclipse.org/git-commit: "@ignore@"
       jkube.eclipse.org/git-branch: "@ignore@"
-      image.openshift.io/triggers: |-
```

Comment:
@AmitPatil05 You haven't removed this correctly. This attribute has multi line scalar value

You need to delete till line 19 (where json array ends)